backend/core/admin_database.py

```
@@ -226,41 +295,614 @@ def _ensure_default_admin_user(self, cursor):
+                        params.append(value)
+
+                if not updates:
+                    return False
+
+                # Always update timestamp
+                updates.append("updated_at = ?")
+                params.append(datetime.now())
+                params.append(category_id)
+
+                # Build and execute query with validated field names only
+                query = f"UPDATE followup_categories SET {', '.join(updates)} WHERE id = ?"
```

Ruff (0.12.2)

Possible SQL injection vector through string-based query construction

(S608)